.storybook/blocks/ComponentDetails.jsx

```diff
@@ -461,14 +318,131 @@ function fetchNpmData(packageName, setnpmData, setIsLoading) {
+			resourceWrapperProps = "rgba(203, 56, 55, 0.1)";
+			break;
+			case "repository": 
+			svgAsset = <GitHubSVG />
+			resourceWrapperProps = "rgba(0, 0, 0, 0.1)"
+			break;
+		case "guidelines": 
+			svgAsset = <AdobeSVG />
+			break;
+		default: 
+			break;
+	}
+
+	return (
+		<ResourceIconWrapper backgroundColor={resourceWrapperProps}>
+			{ svgAsset }
+		</ResourceIconWrapper>
+	)
+};
+
+/**
+ * Converts the linkType to the subtitle text
+ */
+const convertLinkTypeText = (linkType) => {
+	switch(linkType) {
+		case "package": 
+			linkType = "npm"
+			break;
+		case "repository": 
+			linkType = "GitHub"
+			break;
+		case "guidelines": 
+			linkType = "Spectrum website"
+			break;
+		default: 
+			break;
+	}
+	return linkType;
+};
+
+export const ResourceLinkContent = ({data, linkType=["package", "repository", "guidelines"]}) => {
+	const packageJson = data;
+
+	// componentGuidelinesName
+	let packageName = "";
+	let packageLink = "";
+	
+	if (linkType === "package") {
+		// NPM package name and link 
+		packageName = packageJson?.name ?? undefined;
+		packageLink = (packageName && typeof packageName !== "undefined") ? `https://npmjs.com/${packageName}` : false;
+	}
+	
+	else if (linkType === "repository") {
+		// repo name and link 
+		packageName = packageJson?.name ? packageJson?.name.split('/').pop() : undefined;
+		packageLink = (packageName && typeof packageName !== "undefined") ? `https://github.com/adobe/spectrum-css/tree/main/components/${packageName}` : false;
+	}
+
+	else if (linkType === "guidelines") {
+		// checks if the any nested component name is included in the page URL
+		for(let i = 0; i < packageJson?.spectrum?.length; i++) {
+			if (window.location.href.includes(packageJson?.spectrum[i]?.componentName)) {
+
+				// guidelines site name and link
+				packageName = packageJson?.spectrum[i]?.componentName ?? undefined;
+				packageLink = (packageName && typeof packageName !== "undefined") ? packageJson?.spectrum[i]?.guidelinesLink : false;
+				break;
+			} 
+			packageName = packageJson?.spectrum[i]?.componentName ?? undefined;
+			packageLink = (packageName && typeof packageName !== "undefined") ? packageJson?.spectrum[i]?.guidelinesLink : false;
+		}
+	}
+
+	else {
+		console.warn(`Are you sure you mean "${linkType}"? Please use a valid link type instead: "package", "repository", "guidelines"`);
+	}
+
+	return (
+		<>
+			{packageLink ? <>
+					<ResourceLink href={packageLink} rel="noopener" className="sb-unstyled">
+						{iconSvgs(linkType)}
+						<ResourceTextWrapper>
+							<div className="spectrum-Heading spectrum-Heading--sizeXS">View {linkType}</div>
+							<div className="spectrum-Body spectrum-Body--sizeS">{convertLinkTypeText(linkType)}</div>
+						</ResourceTextWrapper>
+					</ResourceLink>
+				</>
+				: ""
+			}
+		</>
+	)
+};
+
+export const ResourceListDetails = ({data}) => {
+	const packageJson = data;
+
+	return (
+		<ResourceSection skipBorder={true} className="sb-unstyled">
```

Comment:
I just noticed that `skipBorder={true}` was here, do we still need it for anything? It doesn't look like we're using this prop as far as I can see.